### PR TITLE
Restructuring

### PR DESCRIPTION
**Pull request by TrueTechy**

### CODE REVIEW

File: kodi_jellyfin/connect.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ##################################################################################################
 
 import logging
+import os
```
Comment (mcarlton00) [inline kodi_jellyfin/connect.py]: I don't see `os` used anywhere in this file.  Is it needed?

File: kodi_jellyfin/connect.py [hunk 2]
```diff
@@ -155,7 +157,7 @@ def select_servers(self, state=None):
 
         state = state or self.connect_manager.connect({'enableAutoLogin': False})
         user = {}
-
+
```
Comment (mcarlton00) [inline kodi_jellyfin/connect.py]: Random extra tab?

File: kodi_jellyfin/connect.py (hunk 1 above)
Comment (TrueTechy) [reply kodi_jellyfin/connect.py]: Was going to leave the code refactoring to another PR but I can remove it if you think it'd be okay

File: kodi_jellyfin/connect.py (hunk 2 above)
Comment (TrueTechy) [reply kodi_jellyfin/connect.py]: ¯\_(ツ)_/¯

File: kodi_jellyfin/connect.py [hunk 3]
```diff
@@ -12,13 +13,14 @@
...
-from jellyfin.core.connection_manager import CONNECTION_STATE
-from jellyfin.core.exceptions import HTTPException
+from jellyfin.connection_manager import CONNECTION_STATE
+from jellyfin.exceptions import HTTPException
 
 ##################################################################################################
 
 LOG = logging.getLogger("JELLYFIN." + __name__)
-XML_PATH = (xbmcaddon.Addon(addon_id()).getAddonInfo('path'), "default", "1080i")
+XML_PATH = (xbmcaddon.Addon(id='plugin.video.jellyfin').getAddonInfo('path'), "default", "1080i")
```
[the remark this answers is not in the document]
Comment (TrueTechy) [reply kodi_jellyfin/connect.py]: This was probably copied from somewhere else when it stopped working on me during the moving. I'll revert it back for now

File: kodi_jellyfin/connect.py (hunk 2 above)
Comment (TrueTechy) [reply kodi_jellyfin/connect.py]: Ah pretty sure this had a debugging line in it at some point and the tabs weren't deleted with the line

**Comment by oddstr13** [comment]
Force-push is due to cleaning up history some, master was merged back into this pr 3 times, among other things.

The only code difference introduced by the force-push is the following;
```diff
diff --git a/context.py b/context.py
index e8f349c6..8e165a68 100644
--- a/context.py
+++ b/context.py
@@ -12,7 +12,7 @@ import xbmcaddon
 #################################################################################################
 
 __addon__ = xbmcaddon.Addon(id='plugin.video.jellyfin')
-__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'kodi_jellyfin')).decode('utf-8')
+__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'jellyfin_kodi')).decode('utf-8')
 
 sys.path.insert(0, __base__)
 
diff --git a/context_play.py b/context_play.py
index 8c9d7583..1bb53e7d 100644
--- a/context_play.py
+++ b/context_play.py
@@ -12,7 +12,7 @@ import xbmcaddon
 #################################################################################################
 
 __addon__ = xbmcaddon.Addon(id='plugin.video.jellyfin')
-__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'kodi_jellyfin')).decode('utf-8')
+__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'jellyfin_kodi')).decode('utf-8')
 
 sys.path.insert(0, __base__)
 
diff --git a/default.py b/default.py
index 94ac956e..f2251509 100644
--- a/default.py
+++ b/default.py
@@ -12,7 +12,7 @@ import xbmcaddon
 #################################################################################################
 
 __addon__ = xbmcaddon.Addon(id='plugin.video.jellyfin')
-__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'kodi_jellyfin')).decode('utf-8')
+__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'jellyfin_kodi')).decode('utf-8')
 
 sys.path.insert(0, __base__)
 
diff --git a/service.py b/service.py
index eb7f4c4b..f807e14f 100644
--- a/service.py
+++ b/service.py
@@ -13,7 +13,7 @@ import xbmcaddon
 #################################################################################################
 
 __addon__ = xbmcaddon.Addon(id='plugin.video.jellyfin')
-__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'kodi_jellyfin')).decode('utf-8')
+__base__ = xbmc.translatePath(os.path.join(__addon__.getAddonInfo('path'), 'jellyfin_kodi')).decode('utf-8')
 
 sys.path.insert(0, __base__)
```

**Pull request merged**